src/generators/jsx/constants.mjs

```diff
@@ -0,0 +1,51 @@
+// Maps Node.js API stability indices (0-3) to UI component stability levels.
+export const STABILITY_LEVELS = [
+  'danger', // (0) Deprecated
+  'warning', // (1) Experimental
+  'success', // (2) Stable
+  'info', // (3) Legacy
+];
+
+// Maps HTML tags to corresponding component names in @node-core/ui-components.
+export const TAG_TRANSFORMS = {
+  pre: 'CodeBox',
+  blockquote: 'Blockquote',
+};
+
+// Maps API heading types to their CircularIcon props.
+export const ICON_SYMBOL_MAP = {
+  event: { symbol: 'E', color: 'red' },
+  method: { symbol: 'M', color: 'red' },
+  property: { symbol: 'P', color: 'red' },
+  class: { symbol: 'C', color: 'red' },
+  module: { symbol: 'M', color: 'red' },
+  classMethod: { symbol: 'S', color: 'red' },
+  ctor: { symbol: 'C', color: 'red' },
+};
+
+// Maps API lifecycle change type identifiers to their human-readable labels.
+export const CHANGE_TYPES = {
```

Comment:
IMO, we should import @node-core/i18n and have these as translation strings. This can be done in a follow-up PR. To be clear, @nodejs/web-infra we shouldn't translate the API docs, but at the very least, navigational and non-main-content markers can be translated.